src/Frontend/src/css/site.scss

```diff
@@ -127,102 +101,97 @@ button.accept-policy {
     background-color: #0d1012;
     height: 100px;
     padding: 1rem;
+    /* Up to xs Breakpoint */
+    @media screen and (max-width: 575.98px) {
+        height: 85px;
+    }
 }
 
 .collapse-wrapper {
     display: flex;
     justify-content: end;
-}
-
-    .collapse-wrapper > div {
+    > div {
         width: 400px;
+        /* Up to sm Breakpoint */
+        @media screen and (max-width: 767.98px) {
+            width: 100%;
+        }
     }
+}
 
 .site-nav-desktop {
     gap: 2rem;
+    .btn-menuburger {
+        background-color: #666666;
+        font-size: 1.75rem;
+        height: 100%;
+        padding: 0.25rem;
+        border: none;
+        &:hover,
+        &:focus {
+            background-color: #525252;
+            color:rgba(0, 0, 0, 0);
+        }
+        &:not(.collapsed) {
```

Comment:
Please uncomment the test for this fix at [tests/Layout.cs](https://github.com/SRCStats/SRCStats/blob/f28cb7c41b4071044b4fbafadc46ee6e05b183d7/tests/Layout.cs#L147-L148).